### DISCUSSION TITLE
Sunbird Ed Forms: Type of questions supported

### DISCUSSION THREAD

**Discussion by sagarihanda** [Sunbird-Ed/Community#61] [General Discussion]
We are looking to deploy Sunbird Ed forms for conducting surveys. The survey will include the following type of questions in addition to the ones mentioned in the [documentation](https://ed.sunbird.org/use-1/independent-libraries/sunbirded-forms). 

1. Upload a file (Image / PDF / CSV)
2. Date picker
3. Number / decimal entry _(with constraints)_

Additionally, we would like the survey to be designed in a manner to allow for conditional grouping of questions - one set of questions to appear if a condition is met as per data entered in the previous set. 

Would these functionalities be available? 

@ChakshuGautam @rahul101001000

**Comment by sharathkashyap** [comment]
@sagarihanda Sunbird Forms can be leveraged to create configured surveys. Some of the extensions like FileInput, Date Picker is not supported. Would be glad if we can receive the contribution in this regard. 

cc: @rajeevsathish @luckynvdu
